### DISCUSSION TITLE
add this app in flathub and snap

### DISCUSSION THREAD

**Discussion by olumolu** [ente-io/ente#2338] [Enhancements]
this will resolve the manually updating issue and easy to install and also the users using immutable systems like steam os and silverblue or aeon can use this on there system thanks for the service.

**Comment by alexbap** [comment]
Flathub would be greatly appreciated. Vanilla OS 2.0 user here.

**Comment by jjardon** [comment]
There is a related discussion about this: https://github.com/ente-io/ente/discussions/587
